test/fw/ptl/utils/plugins/ptl_test_runner.py

```diff
@@ -599,6 +676,11 @@ def startTest(self, test):
             self.__failed_tc_count_msg = True
             raise TCThresholdReached
         timeout = self.__get_timeout(test)
+        param_dict = self.__get_param_dictionary()
+        rv = self.__are_requirements_matching(param_dict, test)
+        if rv is False:
+            self.result.startTest(test)
```

Comment:
I suggest adding a comment here about why we are calling `self.result.startTest(test)` for future (and for other devs) reference